app/_service/filter.service.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@angular/core';
```

Comment:
I think we can probably lose this service and revert to a simpler arrangement as is with the timeline.
Wdyt?